index.js

```diff
@@ -17,13 +18,13 @@ function handleStateChangeOnClient(title) {
   }
 }
 
-var DocumentTitle = React.createClass({
-  displayName: 'DocumentTitle',
-
-  propTypes: {
-    title: React.PropTypes.string.isRequired
-  },
+function DocumentTitle() {
+  React.Component.call(this);
+  this.displayName = 'DocumentTitle';
+}
 
+DocumentTitle.prototype = Object.assign(Object.create(React.Component.prototype), {
```

Comment:
I could be wrong, but [`Object.assign` isn't supported in IE <= 11 according to MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/assign#Browser_compatibility).